nemo/utils/exp_manager.py

```diff
@@ -665,7 +665,11 @@ def configure_loggers(
             wandb_kwargs = {}
         if "name" not in wandb_kwargs and "project" not in wandb_kwargs:
             raise ValueError("name and project are required for wandb_logger")
-        wandb_logger = WandbLogger(save_dir=exp_dir, version=version, **wandb_kwargs)
+
+        # Update the wandb save_dir
+        wandb_kwargs['save_dir'] = wandb_kwargs.get('save_dir') or exp_dir
```

Comment:
Don't do get(save_dir) do get(save_dir, None) is not None else exp_dir. Also don't automatically create dir, wandb should do it for you no ?

Reply:
No, wandb is not creating a new directory if it does not exit. It simply creates a random dir within `/tmp` and starts writing logs there.

Reply:
Sad. Oh well. But still don't use get without a default v

Reply:
Done. Updated.